from0k2bp.tex

```diff
@@ -1307,115 +1346,143 @@
-\end{align*}
-where $G_1', G_2'$ are the 2 components of $\mathbf{G}'$ defined above. Now this multiplication
-has, not accidentally, a specifically useful cancellation: the powers of
- $x$ cancel where the indices of $a, G$ match. For maximum clarity, we visualize
-this again as a matrix:
 \[
-\begin{pmatrix}
+\scalemath{0.78}{
+\begin{pmatrix}  
 a_1G_1 + a_2G_2 & x(a_3G_1 + a_4G_2) & x^2(a_5G_1 + a_6 G_2) & x^3(a_7G_1 + a_8 G_2) & x^4(a_9G_1 + a_{10}G_{2})\\
 x^{-1}(a_1G_3+a_2G_4) & a_3G_3 + a_4G_4 & x(a_5G_3 + a_6G_4) & x^2(a_7G_3 + a_8G_4)& x^3(a_9G_3 + a_{10}G_4 \\
 \dots & \dots & a_5G_5 + a_6G_6 & \dots & \dots \\
 \dots & \dots & \dots & a_7G_7 + a_8G_8& \dots  \\
 x^{-4}(a_1G_9 + a_2G_{10}) & \dots & \dots & \dots & a_9 G_9 + a_{10}G_{10} \\
 \end{pmatrix}
+}
 \]
 
 Note, these are not literally matrices we're looking at, but just a
-visualization of the multiplication of the two terms. We see that, as
-before the inclusion of $x$, the main diagonal is exactly the entirety of $A = a_1G_1 + a_2G_2 + \ldots + a_{10}G_{10}$.
+visualization of the multiplication of the two terms in \eqref{condensing-vector-new-A:1}.
+We see that, as before the inclusion of~$x$, the main diagonal is exactly the entirety of 
+$A = a_1G_1 + a_2G_2 + \ldots + a_{10}G_{10}$.
 
 Now we see, however, the importance specifically of the diagonals, as
 opposed to other slicings-up of the matrix: the diagonals are the sets
 of terms which are multiplied by the same power of $x$. To validate the
 correspondence between $A$ and $A'$, a Verifier will need to check that the
 entire set (``matrix'') of terms; to do this he checks whether:
-\[
-A' = \sum\limits_{\textrm{max}(1, 1-k)}^{\textrm{min}(5, 5-k)}x^k A_k = \sum\limits_{\textrm{max}(1, 1-k)}^{\textrm{min}(5, 5-k)} x^k \textbf{a}_{i+k}\textbf{G}_i \quad \textrm{for }\ k=-4,-3..0..3,4
-\]
-He only checks the left equation of course; the full decomposition is
+\begin{align*}
+A' = \sum\limits_{\textrm{max}(1, 1-k)}^{\textrm{min}(5, 5-k)}x^k A_k 
+   \stackrel{\text{(see }\eqref{condensing-vector-diagonals-A_k:1}\text{)}}{=} 
+   \sum\limits_{\textrm{max}(1, 1-k)}^{\textrm{min}(5, 5-k)} x^k 
+   \textbf{a}_{i+k}\textbf{G}_i \quad \textrm{for }\ k=-4,-3,..,0,..,3,4
+\end{align*}
+Verifier only checks the left equation of course; the full decomposition is
 known only to the Prover.
 
-Imagining this as an interaction; it's a little confusing since it
-\emph{only }has any point in cases where we ``recurse'':
-
+It is a little confusing to imagine this interaction since it has \emph{only} a point in cases where we employ ``recursion'':
 The Prover asserts that $A$ is a commitment to $\mathbf{a}$, and then sends the full
 list $A_{-4}, A_{-3}, \ldots, A_0=A, \ldots, A_{4}$. Then the verifier replies with a challenge $x$. Unlike a typical
-Sigma protocol, the Prover does not then send back a proof. What happens
+Sigma protocol, the Prover does not send back a proof then. What happens
```

Comment:
No, 'does not then send back a proof' is correct (it's a slightly more formal writing style, slightly old fashioned but still common in academic writing, I guess you just aren't used to it).

Reply:
I see. I'll revert.